docs/using/custom-images.md

```diff
@@ -0,0 +1,74 @@
+# Building a custom set of images
+
+This section describes how to build a custom set of images.
+It may be helpful if you need to change Ubuntu or Python version, or make a significant change to the build process itself.
```

Comment:
```suggestion
It may be helpful if you need to change the Ubuntu or Python version, or to make a significant change to the build process itself.
```